README.md

```diff
@@ -31,3 +31,7 @@ trademarks or logos is subject to and must follow
 [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
 Any use of third-party trademarks or logos are subject to those third-party's policies.
+
+## Run and Test
+To run the main use: go run .
```

Comment:
When giving relative path descriptions, let the users know what directory they should be `cd`d into

Reply:
I see, done.

Reply:
I see, done.